vulnerabilities/package_managers.py

```diff
@@ -489,3 +509,75 @@ async def fetch(self, pkg, session):
             pass
 
         self.cache[pkg] = versions
+
+
+class GoproxyVersionAPI(VersionAPI):
+
+    package_type = "golang"
+    pkg_mappings = {}
```

Comment:
Why use `pkg_mappings` ? Also `pkg_mappings` as a name is rather unhelpful. It doesn't convey what kind of mapping this is.

 Since this class inherits `VersionAPI` it has the `cache` attribute for this purpose. Maybe use that ? The `get` method also uses the `cache` for this.

Reply:
This variable maps package name(from github advisory) to module name(variable `trimmed_pkg` in `fetch` method).
I added this mapping because it can't be easilly done in `process_name` method of `github.py`, it  involves trimming url path and making request to proxy.golang to validate the trimmed path.

Reply:
Oh I see. Then maybe rename this to something like `module_name_by_package_name` ?